### PR TITLE
LPS-198741 Adapt styles

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/JournalConfigurationAction.java [hunk 1]
```diff
@@ -152,14 +152,16 @@ public void processAction(
...
-		validateEmail(actionRequest, "emailArticleAdded");
-		validateEmail(actionRequest, "emailArticleApprovalDenied");
-		validateEmail(actionRequest, "emailArticleApprovalGranted");
-		validateEmail(actionRequest, "emailArticleApprovalRequested");
-		validateEmail(actionRequest, "emailArticleExpired");
-		validateEmail(actionRequest, "emailArticleReview");
-		validateEmail(actionRequest, "emailArticleUpdated");
-		validateEmailFrom(actionRequest);
+		if (!FeatureFlagManagerUtil.isEnabled("LPS-197692")) {
+			validateEmail(actionRequest, "emailArticleAdded");
```
Comment (ruben-pulido) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/JournalConfigurationAction.java:156]: Should we perform individual validations instead, depending on which navigation value was selected?

Comment (jkappler) [reply modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/JournalConfigurationAction.java:156]: Yes, I believe so @ruben-pulido, but better in another task in the story

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**